libs/agno/agno/team/team.py

```diff
@@ -4054,15 +4063,16 @@ def _calculate_full_team_session_metrics(self, messages: List[Message]) -> Metri
 
         # Get metrics of the team-agent's messages
         for member in self.members:
-            # Only members with memory
-            if member.memory is not None:
-                if member.memory.runs is not None:
-                    for runs in member.memory.runs.values():
-                        for run in runs:
-                            if run is not None and run.messages is not None:
-                                for m in run.messages:
-                                    if m.role == assistant_message_role and m.metrics is not None:
-                                        current_session_metrics += m.metrics
+            # Only members with agent_session
+            if member.agent_session is not None:
```

Comment:
But members won't have sessions. We should rather use the member runs that we keep track of?

Reply:
Or actually since this is across the whole session, we'll have to pull up all the runs in the session and add it all up... that might be a DB function rather. 

This one might be a problem